src/PrestaShopBundle/Resources/config/services/adapter/tax_rules_group.yml

```
@@ -2,6 +2,22 @@ services:
   _defaults:
     public: true
 
+  prestashop.adapter.tax_rules_group.command_handler.add_tax_rules_group_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler\AddTaxRulesGroupHandler'
+    arguments:
+      - '@prestashop.adapter.tax_rules_group.repository.tax_rules_group_repository'
+    tags:
+      - name: 'tactician.handler'
+        command: 'PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Command\AddTaxRulesGroupCommand'
+
+  prestashop.adapter.tax_rules_group.command_handler.edit_tax_rules_group_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler\EditTaxRulesGroupHandler'
+    arguments:
+      - '@prestashop.adapter.tax_rules_group.repository.tax_rules_group_repository'
+    tags:
+      - name: 'tactician.handler'
+        command: 'PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Command\EditTaxRulesGroupCommand'
```

```suggestion
  PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler\EditTaxRulesGroupHandler:
    autowire: true
    tags:
      - name: 'tactician.handler'
        command: 'PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Command\EditTaxRulesGroupCommand'
```
Actually, you would even be able to do this (that's the advantage of the FQCN definitions)

---

Fixed